### DISCUSSION TITLE
Theme Doesn't Save All The Time

### DISCUSSION THREAD

**Discussion by stondino00** [open-webui/open-webui#5523] [General]
Doesn't appear changing the theme when a user logins sticks.  Set to dark and saved but logging in again shows white theme.

**Comment by tjbck** [comment]
Unable to reproduce, let us know your findings!

**Reply by github-pmj** [reply]
I have the same issue - as per my comment below, I have a local virtual install and this still occurs in 0.3.23

**Reply by github-pmj** [reply]
Just noticed that the Min P setting in System / Advanced also fails to save - now on V3.30

**Reply by github-pmj** [reply]
3rd June 2025 - This is still an issue with the theme in version v0.6.13. I was hopeful that when the min_p issue was fixed that the theme would be too but no such luck.

**Comment by github-pmj** [comment]
I have the same issue. Other settings, e.g. Notifications, System Prompt, persist but theme does not (other than in browser cache). When I click Save I get the 'successful' message and I can see the webui.db timestamp update. This is on Windows 10 with a local install (.venv) of 0.3.23
`py -m pip install --upgrade open-webui`

Not sure whether it's relevant but inspecting webui.db shows the config table is empty and I have no config.json file either.

28th Sep. v3.30 - The Min P setting in System / Advanced also fails to save

**Comment by licaon-kter** [comment]
Any news? Still a bug in 0.6.18, neither theme nor locale are saved per account :cry: 

Why is this a Discussion instead of an Issue?
